docs/ui/components/Table/Row.tsx

```diff
@@ -0,0 +1,19 @@
+import { css } from '@emotion/react';
+import { theme } from '@expo/styleguide';
+import React, { PropsWithChildren } from 'react';
+
+type RowProps = PropsWithChildren<object>;
+
+export const Row = ({ children }: RowProps) => <tr css={tableRowStyle}>{children}</tr>;
+
+const tableRowStyle = css({
+  borderBottomWidth: 1,
+  borderBottomStyle: 'solid',
+  borderBottomColor: theme.border.default,
+  '&:last-child': {
+    borderWidth: 0,
+  },
```

Comment:
This probably also can be removed. I remember talking with Jon about differentiating every second row, and border on rows were not a case than IIRC. The rows also doesn't have the border on the latest design which Jon have provided.

Reply:
Do you mean the full row styling can be removed? Or jus the last child part?

Reply:
All selected rows, they are there just for the border. To implement the design we only need `&:nth-of-type(2n)` part.

Reply:
Awesome, just did that, I think we are good to go then 😄

Reply:
Just spotted that, this style probably should be moved to `TableHead`, because it uses `Row` internally and table heading should have this border, sorry about that!

Reply:
No worries! Let me check if we can apply that, or if we need to do some props spread to do that.

Reply:
Done!